edge/services/helloworld/MaintainHW.md

```diff
@@ -0,0 +1,18 @@
+# Process for the Horizon Development Team to Make Updates to the Helloworld Service
+
+Instructions for the open-horizon developers that are maintaining this example code:
+
+- Do the steps in the [README.md Preconditions](#preconditions) section, **except**:
```

Comment:
We cannot say "Do", try Perform or complete.

Reply:
done